lib/include/kframework/avm/teal/teal-syntax.md

```diff
@@ -451,6 +451,7 @@ We define the syntax of TEAL's comments (using K's built-in sort `#Layout`), alo
   syntax Label           ::= r"({AlnumUbar}|{Special})+" [token]
   syntax HexToken        ::= r"0x{HexDigit}+"            [prec(2),token]
   syntax TAddressLiteral ::= r"[0-9A-Z]{58}"             [prec(1),token]
+  syntax #UpperID ::= TAddressLiteral [token]
```

Comment:
Why is this needed?

Reply:
I was running into a lexing issue with address literals which this seemed to fix. However, I can't remember the details, so I deleted it to see if it is still needed.

Reply:
It seems to work fine without.